lib/web_ui/lib/src/engine/text/line_breaker.dart

```diff
@@ -91,7 +97,8 @@ bool _hasEastAsianWidthFWH(int charCode) {
 /// * https://www.unicode.org/reports/tr14/tr14-45.html#Algorithm
 /// * https://www.unicode.org/Public/11.0.0/ucd/LineBreak.txt
 LineBreakResult nextLineBreak(String text, int index) {
-  LineCharProperty? curr = _normalizeLineProperty(lineLookup.find(text, index));
+  int? codePoint = getCodePoint(text, index);
+  LineCharProperty? curr = _normalizeLineProperty(codePoint);
```

Comment:
Now that `_normalizeLineProperty` does not return null any more, can `curr` become non-null?

Reply:
Good catch! Is the analyzer supposed to warn about this?